db/migrate/20230120170643_create_like.rb

```diff
@@ -1,8 +1,10 @@
 class CreateLike < ActiveRecord::Migration[7.0]
-  def change
-    create_table :likes do |t|
+   def change
+      create_table :likes do |t|
+         t.datetime :CreatedAt
+         t.datetime :UpdatedAt
 
-      t.timestamps
-    end
-  end
+         t.timestamps
```

Comment:
- [ ] `t.timestams` will crate updated_at and created_at columns, which means that you don't need to create extra columns, please 🙏🏽  remove UpdateAt and CreateAt columns to avoid future conflicts.